boards/arm/secureiot1702/doc/secureiot1702.rst

```diff
@@ -0,0 +1,82 @@
+.. _secureiot1702:
+
+Microchip SecureIoT1702
+#######################
+
+Overview
+********
+
+Board configuration for Microchip CEC1702 demo board. This is a cryptographic
+embedded controlled based on an ARM Cortex-M4.
+
+Highlights of the board:
+
+- CEC1702 32-bit ARM® Cortex®-M4F Controller with Integrated Crypto
+- Compact, high-contrast, serial graphic LCD Display Module with back-light
+- 2x4 matrix of push buttons inputs
+- USB-UART Converter as debug interface
+- Potentiometer to ADC channel
+- Serial Quad I/O (SQI) flash
+- OTP programmable in CEC1702
+- Two expansion headers compatible with MikroElektronika mikroBUS™ Expansion interface
+
+More information can be found from `SecureIoT1702 website`_ and
+`CEC1702 website`_. The SoC programming information is available
+in `CEC1702 datasheet`_.
+
+Supported Features
+==================
+
+Following devices are supported:
+
+- Nested Vectored Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+- Serial Ports (NS16550)
+- Quad Master-only SPI controller (QMSPI)
+
+
+Connections and IOs
+===================
+
+Please refer to the `SecureIoT1702 schematics`_ for the pin routings.
+Additional devices can be connected via mikroBUS expansion interface.
+
+Programming and Debugging
```

Comment:
Add a Building section that includes an example of building one of the sample apps, as done in most other board documents.

Reply:
@fabled Will you be adding a building section to the document?

Reply:
@dbkinder Yes, I'll take most of what is in `mec2016evb_assy6797`and/or `mec15xxevb_assy6853` since they are essentially the same CPU. Though, perhaps I try to get my python image generator script integrated also to avoid dependency on the Microchip Windows tool. I am busy for this and next week. But will try to get this polished before end of this month - perhaps even this week if I get a good timeslot.

Reply:
Please re-request my review when the building section is here.

Reply:
Added small section here based on four other similar docs. The building is also already described in more detail a bit below.